test/SignalR.Orleans.Tests/SignalR.Orleans.Tests.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="5.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="6.0.3" />

```

Comment:
Not sure if the build will pass since this is using 6.0.3 and the docker image SDK version is 6.0.2 (for now leave it as is and we'll see)

Reply:
should be fine, since its a nuget